spec/lib/reporting/identity_verification_report_spec.rb

```diff
@@ -4,135 +4,223 @@
-
-        # Fraud review user (incomplete)
-        { 'user_id' => 'user3', 'name' => 'IdV: doc auth welcome visited' },
-        { 'user_id' => 'user3', 'name' => 'IdV: doc auth welcome submitted' },
-        { 'user_id' => 'user3', 'name' => 'IdV: doc auth image upload vendor submitted', 'success' => '1' },
-        { 'user_id' => 'user3', 'name' => 'IdV: final resolution', 'fraud_review_pending' => '1' },
-        { 'user_id' => 'user3', 'name' => 'Fraud: Profile review passed', 'success' => '1' },
-
-        # Success through address confirmation user
-        { 'user_id' => 'user4', 'name' => 'IdV: GPO verification submitted' },
-
-        # Success through in-person verification, failed doc auth (rejected)
-        { 'user_id' => 'user5', 'name' => 'IdV: doc auth welcome visited' },
-        { 'user_id' => 'user5', 'name' => 'IdV: doc auth welcome submitted' },
-        { 'user_id' => 'user5', 'name' => 'IdV: doc auth image upload vendor submitted', 'doc_auth_failed_non_fraud' => '1' },
-        { 'user_id' => 'user5', 'name' => 'IdV: final resolution', 'in_person_verification_pending' => '1' },
-        { 'user_id' => 'user5', 'name' => 'GetUspsProofingResultsJob: Enrollment status updated' },
-
-        # Incomplete user
-        { 'user_id' => 'user6', 'name' => 'IdV: doc auth welcome visited' },
-        { 'user_id' => 'user6', 'name' => 'IdV: doc auth welcome submitted' },
-        { 'user_id' => 'user6', 'name' => 'IdV: doc auth image upload vendor submitted', 'doc_auth_failed_non_fraud' => '1' },
-      ],
+      fetch: cloudwatch_results,
     )
 
     allow(report).to receive(:cloudwatch_client).and_return(cloudwatch_client)
   end
-  # rubocop:enable Layout/LineLength
+
   describe '#as_csv' do
     it 'renders a csv report' do
-      expected_csv = [
-        ['Report Timeframe', "#{time_range.begin} to #{time_range.end}"],
-        ['Report Generated', Date.today.to_s], # rubocop:disable Rails/Date
-        ['Issuer', issuer],
-        [],
-        ['Metric', '# of Users'],
-        [],
-        ['Started IdV Verification', 5],
-        ['Submitted welcome page', 5],
-        ['Images uploaded', 5],
-        [],
-        ['Workflow completed', 4],
-        ['Workflow completed - Verified', 1],
-        ['Workflow completed - Total Pending', 3],
-        ['Workflow completed - GPO Pending', 1],
-        ['Workflow completed - In-Person Pending', 1],
-        ['Workflow completed - Fraud Review Pending', 1],
-        [],
-        ['Successfully verified', 4],
-        ['Successfully verified - Inline', 1],
-        ['Successfully verified - GPO Code Entry', 1],
-        ['Successfully verified - In Person', 1],
-        ['Successfully verified - Passed Fraud Review', 1],
-      ]
-
       aggregate_failures do
-        report.as_csv.zip(expected_csv).each do |actual, expected|
+        report.as_csv.zip(JSON.parse(expected_as_csv.to_json)).each do |actual, expected|
```

Comment:
what is the purpose of JSON-ifying and de-JSONifying these?

Reply:
Great question! I think I was debugging something and that got left in. Removed in 6097b0af2